queries.sql

```diff
@@ -58,4 +58,127 @@ FROM
     animals
 WHERE
     (weigth_kg >= 10.4)
-    AND (weigth_kg <= 17.3);
\ No newline at end of file
+    AND (weigth_kg <= 17.3);
+
+begin;
+
+UPDATE
+    animals
+SET
+    species = 'unspecified';
+
+select
+    *
+from
+    animals;
+
+rollback;
+
+select
+    *
+from
+    animals;
+
+begin;
+
+UPDATE
+    animals
+SET
+    species = 'digimon'
+WHERE
+    name LIKE '%mon';
+
+select
+    *
+from
+    animals;
+
+UPDATE
+    animals
+SET
+    species = 'pokemon'
+WHERE
+    species IS NULL
+    OR species = '';
+
+select
+    *
+from
+    animals;
+
+commit;
+
```

Comment:
- [ ] Nice work implementing the transactions :+1: However the questions about **delete** (see image) are not answered, kindly make sure you answer the two remaining queries to the `queries.sql` file to follow the project requirement.
![Screenshot from 2023-09-28 11-30-45](https://github.com/ZabihullahNooriWardak/vet-Clinic/assets/91142432/478e352d-e86e-4853-a181-ff8f1232a136)